CONTRIBUTING.md

```diff
@@ -165,4 +166,16 @@ Please keep in mind that the team may **suspend or reject** pull requests. There
 -   Poor quality code.
 -   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.
 
-Please also note that we do not merge pull requests until the changes are documented.
+Please also note that we do not merge pull requests until the changes are documented. You're welcome to document a new capability yourself. You should add your description as a pull request comment, and include links to Testcafe [documentation](https://testcafe.io/documentation) pages that need editing. While writing the documentation, please, follow the [guidelines](#guidelines).
+
+## Contribute to Documentation
+
+Testcafe [documentation](https://testcafe.io/documentation) is not open-source. If you find an inaccuracy in the docs, send us a note in a [Github issue](https://github.com/DevExpress/testcafe/issues/new). Please, don't forget to add `AREA: docs` label so that the appropriate team gets a new issue for review.
```

Comment:
* The name of our product should always be written in UpperCamelCase, unless we're explicitly talking about the `testcafe` npm package.
* The first sentence states a fact that seemingly contradicts the heading, but there's no follow-up. Why is it important that our docs aren't open-source?
* Github issues aren't a communication platform. You can't send notes in issues. Both you and the issue can, however, *notify* us of something. 
```suggestion
Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.
```